tests/test_persister.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import pytest
+
+from tests.mocks import MockPersisterIO
+from ttt.game.board import Board
+from ttt.game.game import Game
+from ttt.persister.persister import Persister
+from ttt.players.simple_computer_player import SimpleComputerPlayer
+
+
+@pytest.fixture
+def json_data():
```

Comment:
Does this name reflect what the `json_data` is? How could you make this more descriptive?

Reply:
Changed to saved_game_json_data